dashboard.json

```diff
@@ -2083,7 +2083,12 @@
               "key": "station",
               "operator": "=~",
               "value": "/^$Station$/"
-            }
+            },
+            {
+              "condition": "AND",
+              "key": "module",
+              "operator": "=~",
+              "value": "/^$Module$/"
```

Comment:
@florianbeer @dkozinn I think, this broke the JSON because the `{` is missing here

Reply:
It would appear that you are correct. I have no idea how that happened,